frontend/packages/helm-plugin/src/components/forms/ProjectHelmChartRepository/CreateProjectHelmChartRepositoryForm.tsx

```diff
@@ -0,0 +1,126 @@
+            helpText={t('helm-plugin~A display name for the Helm Chart repository.')}
+            required
+          />
+          <InputField
+            type={TextInputTypes.text}
+            name="repoDescription"
+            label={t('helm-plugin~Description')}
+            helpText={t('helm-plugin~A description for the Helm Chart repository.')}
+          />
+          <CheckboxField
+            name="disabled"
+            label={t('helm-plugin~Disable usage of the repo in the namespace')}
+          />
+          <InputField
+            type={TextInputTypes.text}
+            name="repoUrl"
+            label={t('helm-plugin~URL')}
+            helpText={t('helm-plugin~Helm Chart repository URL.')}
+            required
+          />
+          <ExpandCollapse
+            textExpanded={t('helm-plugin~Hide advanced options')}
+            textCollapsed={t('helm-plugin~Show advanced options')}
+          >
+            <FormSection>
+              <p className="pf-c-form__helper-text">
+                {t(
+                  'helm-plugin~Add credentials and custom certificate authority (CA) certificates to connect to private helm chart repository',
+                )}
+              </p>
+              <ResourceDropdownField
+                name="ca"
+                label={t('helm-plugin~CA certificate')}
+                resources={[
+                  {
+                    isList: true,
+                    kind: ConfigMapModel.kind,
+                    namespace: 'openshift-config',
+                    optional: true,
+                    prop: ConfigMapModel.id,
+                  },
+                ]}
+                dataSelector={['metadata', 'name']}
+                fullWidth
+                placeholder={t('helm-plugin~Select ConfigMap')}
+                showBadge
+                autocompleteFilter={autocompleteFilter}
+              />
+
+              <ResourceDropdownField
+                name="tlsClientConfig"
+                label={t('helm-plugin~TLS Client config')}
+                resources={[
+                  {
+                    isList: true,
+                    kind: SecretModel.kind,
+                    namespace: 'openshift-config',
+                    optional: true,
+                    prop: SecretModel.id,
+                  },
+                ]}
+                dataSelector={['metadata', 'name']}
+                fullWidth
+                placeholder={t('helm-plugin~Select Secret')}
+                showBadge
+                autocompleteFilter={autocompleteFilter}
+              />
+            </FormSection>
+          </ExpandCollapse>
+        </FormSection>
+      </FormBody>
+      <FormFooter
+        handleReset={handleReset}
+        errorMessage={status?.submitError}
+        isSubmitting={isSubmitting}
+        submitLabel={t('helm-plugin~Create')}
+        disableSubmit={isSubmitting || !_.isEmpty(errors)}
+        resetLabel={t('helm-plugin~Cancel')}
```

Comment:
Clicking on Cancel button, enables the `Submit` button even though the required fields are not filled.

Reply:
Fixed it